src/w3d/math/gamemath.cpp

```diff
@@ -34,6 +37,31 @@ const Array<float, ARC_TABLE_SIZE> _FastAsinTable = calculateFastTable<float, AR
 const Array<float, SIN_TABLE_SIZE> _FastSinTable = calculateFastTable<float, SIN_TABLE_SIZE>(GameMath::Sin, false);
 const Array<float, SIN_TABLE_SIZE> _FastInvSinTable = calculateFastTable<float, SIN_TABLE_SIZE>(GameMath::Inv_Sin, false);
 
-void GameMath::Init() {}
+void GameMath::Init()
+{
+#ifdef GAME_DLL
+    // This is here to initialize the game version of the fast tables since we don't yet own everything that uses them.
+    float *table = (float *)PICK_ADDRESS(0x00A41500, 0x00E10BA8);
+    for (int i = 0; i < _FastAcosTable.Size(); i++) {
+        table[i] = _FastAcosTable[i];
+    }
+
+    table = (float *)PICK_ADDRESS(0x00A3F500, 0x00E0EBA8);
+    for (int i = 0; i < _FastAsinTable.Size(); i++) {
+        table[i] = _FastAsinTable[i];
+    }
+
+    table = (float *)PICK_ADDRESS(0x00A3E500, 00E0DBA8);
```

Comment:
for wb address missing 0x for lower one too

Reply:
fixed.